src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -1,10 +1,6 @@
 import findLastIndex from 'ramda/src/findLastIndex';
 import propSatisfies from 'ramda/src/propSatisfies';
 import path from 'ramda/src/path';
-import nodeLogger from '#lib/logger.node';
-import { RADIO_SCHEDULE_DATA_INCOMPLETE_ERROR } from '#lib/logger.const';
-
-const logger = nodeLogger(__filename);
 
 export const getProgramState = (currentTime, startTime, endTime) => {
```

Comment:
Oops these changes have been reverted. I think I hit the update branch button which has overwritten the changes. We need to fix before we merge